nano/styles/_icon.less

```diff
@@ -11,7 +11,7 @@
     background-image: url(uiLinkPendingIcon.gif);
 }
 
-.linkPending .uiIcon16 {
+.linkPending .fa {
```

Comment:
this actually means "only affect when `.linkPending` *and* `.fa` are present" (IE, remove icons when showing the linkPending scrollthing)